### ISSUE TITLE
Touch base: What do you think this WG should be, what should it produce, and what should it achieve?

### ISSUE THREAD

**Issue by bnb** [nodejs/evangelism#14]
The discussions in this WG have been a bit lop-sided in my eyes. I feel like everybody has their ideas of what this WG is, does, produces, achieves, etc. Everybody is looking writing against their vision or idea of the WG, but nobody has really defined outright what we're setting out to achieve in a measurable manner. 

I'd like to know what everyone thinks about this--what are our exact goals? If you're willing, would you try to come up with a few, even if they don't come to you immediately? To help this, maybe think about what evangelism is in terms of software, and what it means to practice it.

Here are a few I have in mind:
- Bring a strong social media presence that reflects positively on io.js.
- Develop a unique writing style that anybody writing for io.js can use.
- Through various media, bring in a noticeable amount of users to at least give io.js a shot.
- Facilitate teaching of the actual piece of software called io.js, connecting community with experts in a way that works for those trying to learn.
- Facilitate education about the principles behind io.js, and how its community is a part of that
- Actively seek to involve other projects that wouldn't have supported, converted, or started without encouragement

Again, try to think of and respond with at least a few ideas of your own. If you have any responses to the ones I wrote, I'd really love to hear about those thoughts and talk about them.

**Comment by mikeal** [comment]
Thanks for kicking off this discussion. Keep in mind that if we end up changing the mission sufficiently we'll want to amend the charter https://github.com/iojs/io.js/blob/v1.x/WORKING_GROUPS.md#evangelism

**Comment by mikeal** [comment]
Rather than begin with the type of work we do I think that we should create a strong mission statement and from there we can iterate on the work we can do to best accomplish that mission. The current mission, which I'm not terribly attached to even though I wrote it is:

> The evangelism working group promotes the accomplishments of io.js and lets the community know how they can get involved.

I'm struggling with how best to describe my ideal vision statement but I think it will help if I can iterate a few of the things I think that we've done well so far and their effect.

### Brought greater transparency of io.js to the community

Doing things in the open on GitHub and publicly recorded meetings isn't enough. Most developers don't have the time to watch a whole video or ready every email notification for everything that happens on GitHub. The weekly rollups and what we put out on twitter does a lot to inform the developer community about we're doing.

### Brought greater developer engagement to io.js

We've successfully used our social media following to attract people to specific issues of interest where we need wider participation. The roadmap is a big success story here, we were able to get input from a much larger group of developers than ever before. In addition, whenever we spin up a new working group or any new initiative social media has been critical to attracting people. The i18n effort never would have taken off the way it did if it wasn't for social media.

### Created compelling project messaging

This we could have done better at. The first thing I did was write up the messaging for the 1.0 release, which still stands and is ES6 centric. The next challenge is messaging the roadmap and while I'm optimistic the work isn't done there :)

**Comment by bnb** [comment]
Okay, I like the idea of creating a mission statement and iterating on it. I'm breaking my thoughts up into more consumable comments, as there's a lot to say. Huge apologies in advance for the volume.

I like your list of what we've already done. I agree with it wholly. In fact, I think you've covered all the points that we have succeeded at, covered the whole picture, which I have come to understand that you are pretty good at, especially when others haven't. I can't really think of any more points, so I think we need to focus on these, and improve upon them - I know we can exponentially grow our success.

**Comment by bnb** [comment]
My thoughts on specific points you made:

#### Public Availability of Information

I definitely agree that staying on GitHub isn't enough. An idea I have is to post meeting minutes more publicly and visibly, in full and abbreviated editions. It would not be a huge amount of work to clean up the presentation of the TC/WGs' meetings' minutes, and publish them in a place where they are highly visible. This way, people don't have to watch the entire meeting to know what was talked about and what was decided. 

I think this greatly furthers transparency by lowering the barrier to entry. It's information that's not going to be converting people to use io.js, but it will provide a place for them to start learning about what's going on with io.js in any of the WGs, and know where to start once they're ready to engage. 

#### Public Engagement

As I said, it's great to know that the weekly posts on Medium have pretty much provided the foundation for community engagement. I think we need to start building off of that - branching out from there. Here are some points I think we can use to engage more overall: 

#### Broadcast more:

Being more vocal about what's happening around the community will help bring people in. This could take the form of posting **links** to open discussions of the WGs, **frequent news** updates about the status of different parts of the system, and having **discussions focused on non-participants and outsiders**. 

###### Sharing Links Consistently

Links to WG topics that are open for input and discussion (and that won't be solved quickly) are an _extremely_ easy source of content. The actual content exists already, we're just providing a path for people to see it and participate when they otherwise wouldn't.

###### Keeping public knowledge current in public places

News updates are a _must_.. I didn't think about it until just now, but I realized the 1.4.1 release wasn't covered by an official account that I saw except for a retweet, and the whole thing with 1.4.1's Windows TLS security problem wasn't discussed on official accounts that I saw. 

Those types of news are important to make the community aware of - even though the latter may seem negative to the image of io.js, it can actually be turned into something positive. Make the community aware, give them frequent updates on what's going on, and give them a path to contribute. Stuff like this only turns bad when it is handled poorly or not at all. If you're up front and blunt with your information, there's not a problem.

(Do note: I didn't mean this as a judgement on anybody, their ability to do anything, or the quality of anything they did. I think everybody's doing a great job so far, and I think we can improve upon it.)

###### Engage the non-community

There's a way to directly engage people who use io.js, but don't participate in the official community, and people who haven't taken the leap to start using io.js yet. That way is to create content exclusively for them. An example I can think of is NodeSchool, where they have some workshops that are meant to introduce Node to people who haven't ever used it before. They're frequently engaging people who likely wouldn't have gotten into Node if they hadn't taken the simple step of signing up for a short session. 

This is the type of thing we can do with io.js. There are ways to do it online, I'm sure, and we should try to come up with them. One good way to do this is to ask the active community what would have been useful to them when they were getting engaged with us, what would have brought them in sooner or made it easier for them to enter.

**Comment by bnb** [comment]
#### WG Spin-ups and engagement

It's awesome to hear that the WG spin-ups have been largely dependent on social media--I wasn't aware of that. I also came in literally the day after the i18n stuff blew up, so I wasn't 100% aware of how critical social media was to that, either.

I think we need to develop this further - come up with a strategy for continuing to spark interest in new additions, and further develop interest in existing ones. A laid out plan for what to do in each case would help us generate interest more consistently. The main things these would need to include are: 
- Spark interest in additions
  - Announce the addition, with information on how to contribute immediately
  - Include content (quotes, reposts, answers to common questions) about the addition from the people who are in charge of it
  - Maintain non-annoying reminders about the addition for a period of time that will ensure everybody who could see it does
- Further develop interest in existing group
  - On a consistent but infrequent basis, remind the community and prompt them to start helping
  - Make posts about important or relevant events (issues, discussions, releases, etc.) within the existing group
  - Bring in content from existing members (such as reposts or links to blog posts, that are linked with them or their accounts) that is relevant to the existing group

**Comment by bnb** [comment]
#### Messaging:

I'd definitely say messaging is important to get right. Can you share what your ideas for messaging are in terms of the roadmap?

**Comment by snostorm** [comment]
Ping @iojs/website as this discussion affects both groups in many way.

Re: content creation, we should be working together to ensure that the site is moving forward in a way that facilitates the "open governance" communication goals both our teams share.

Re: WG meetings, I think it will be important to have some cross-pollination between the website, evangelism and i18n content groups. We should probably have at least one "representative" (or dual members) on WG meetings as a general goal to make sure we're all in the loop with each other's efforts.

Thoughts?

**Comment by bnb** [comment]
@snostorm I definitely agree that cross-pollination is going to be something that will benefit all three groups.

I can't really think of a less-blunt way to say this, so apologies, but what would be an example of something the Evangelism WG would help the Website WG create/develop/maintain, and vice versa?

**Comment by snostorm** [comment]
@bnb off the top of my head:
- "Add Social Media Style Guide" via https://github.com/iojs/evangelism/pull/6 has high overlap with the general "content contribution" guidance I was looking for us to add on the website project.
-  To quote you: "An idea I have is to post meeting minutes more publicly and visibly ... It would not be a huge amount of work to clean up the presentation of the TC/WGs' meetings' minutes, and publish them in a place where they are highly visible. " Well, to me, iojs.org has the potential to be one of the go-to resources for the community. Thus, it would be highly visible to be included there. (Possibly other places as well.) Same for our Weekly Update posts, etc.
- There also seems to be some confusion relating to logos/branding/etc. which will overlap the two groups especially in regard to evolving the website design. (And how does that translate to how we look on our social media outlets, etc.)

To be more concise, you should consider iojs.org one of your many publishing platforms. As your platform, we want to make it as easy as possible for your group to publish content there once it is ready.

**Comment by a0viedo** [comment]
> Brought greater transparency of io.js to the community
> Doing things in the open on GitHub and publicly recorded meetings isn't enough. Most developers don't have the time to watch a whole video or ready every email notification for everything that happens on GitHub. The weekly rollups and what we put out on twitter does a lot to inform the developer community about we're doing.

I think this should be important. For someone who doesn't watch the TC meeting videos nor the WG meetings videos, there should be a way to communicate what we are doing.

> News updates are a must.. I didn't think about it until just now, but I realized the 1.4.1 release wasn't covered by an official account that I saw except for a retweet, and the whole thing with 1.4.1's Windows TLS security problem wasn't discussed on official accounts that I saw.

Noticed the same thing. And even with other official announcements happened the same. I think there should be a clear workflow on how to share announcements/news on social media. Ultimately, others WGs would say when an achievement is made: "we should tell this to **evangelism**".

**Comment by bnb** [comment]
@snostorm Is there anything currently that can be done to further this, or has it been taken care of?

**Comment by snostorm** [comment]
No movement at all really. With the growing "family" (aka merger) I think some of the thoughts outlined here will be more important than ever, especially to communicate the governance evolution in the Node.js project.

Over the next few Website team meetings we'll be discussing strategies around content, etc. so it would be great to get some Evangelism ears on those talks. By the same note has anybody from the Website group been attending the meetings here? (Have they been happening? Will check.)

**Comment by mikeal** [comment]
This overlaps with what i just said in another thread but we also need to figure out the right scope of this WG and where the marketing committee can pick up some of the slack -- as well as a good collaboration model with the foundation's marketing committee.

**Comment by bnb** [comment]
@snostorm I'll ping you in a while, once I'm able to sit down and work on io.js consistently (job hunting right now, taking up most of my time). I'd be happy to join that discussion as a Evangelism WG spokesperson, and also start being active in the Website WG again. 

As you probably know, @mikeal would also probably be an ideal spokesperson for the Evangelism WG, too. If anyone else in @nodejs/evangelism would like to take part in the Website WG discussions as a member in a Hangout, just ask!
